core/src/main/java/hudson/model/AbstractBuild.java

```diff
@@ -733,7 +734,9 @@ protected final boolean performAllBuildSteps(BuildListener listener, Iterable<?
 
         private void reportError(BuildStep bs, Throwable e, BuildListener listener, boolean phase) {
             String msg = "Publisher " + bs.getClass().getName() + " aborted due to exception";
-            e.printStackTrace(listener.error(msg));
+            if (!(e instanceof AbortException)){
+                e.printStackTrace(listener.error(msg));
+            }
             LOGGER.log(WARNING, msg, e);
```

Comment:
I'm not talking about the build log (listener) but the Jenkins log Logger.log(..) which is called outside the if() statement